internal/routing/router.go

```diff
@@ -95,6 +96,7 @@ func InitRouter(databaseMgr managers.DatabaseMgr, mailMgr managers.MailMgr, jwtM
 		r.Post("/", subscriptionHdl.Subscribe)
 		r.Delete("/{subscriptionId}", subscriptionHdl.Unsubscribe)
 		r.Get("/{username}", subscriptionHdl.HandleGetSubscriptions)
+		r.Get("/", subscriptionHdl.HandleGetSubscriptions)
```

Comment:
```suggestion
```

Für was ist hier diese Route? Es wird ja ein username benötigt um eine Subscription aufzurufen.

Reply:
Wenn die nicht da ist get ein Get request ohne username an den post endpunkt und gibt einen 405 zurück. Laut Postman muss ein get request ohne username aber ein 400 zurückgeben. Ob der Username dann leer ist wird in HandleGetSubscription geprüft

Reply:
Was hat das aber mit der Route zu tun? Wenn du bei `r.Get("/{username}", subscriptionHdl.HandleGetSubscriptions)` kein usernamen angibst, wirst du trotzdem über diese Route weitergeleitet. Das `{username}` ist hier nur optional

Reply:
ich wurde da an die post weitergeleitet mit einem 405

Reply:
Kannst das rausnehmen, wir lassen den Fehlerfall raus haben wir beschlossen